From c8333f8c7a4e4b9549abeef7530b2cd20a18e537 Mon Sep 17 00:00:00 2001
From: rofl0r <retnyg@gmx.net>
Date: Mon, 12 Oct 2015 12:57:09 +0100
Subject: [PATCH] asound.h: include <time.h> to get struct timespec prototype

without including it, we get
In file included from mixer.c:44:0:
include/sound/asound.h:337:18: error: field 'trigger_tstamp' has incomplete type
include/sound/asound.h:338:18: error: field 'tstamp' has incomplete type
etc.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Patch status: upstream commit c8333f8c7a4e

 include/sound/asound.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/sound/asound.h b/include/sound/asound.h
index a041628ec28e..7c6de81673f5 100644
--- a/include/sound/asound.h
+++ b/include/sound/asound.h
@@ -12,6 +12,7 @@
 #ifndef __SOUND_ASOUND_H
 #define __SOUND_ASOUND_H
 
+#include <time.h>
 #include <linux/types.h>
 
 #define SNDRV_PROTOCOL_VERSION(major, minor, subminor) (((major)<<16)|((minor)<<8)|(subminor))
-- 
2.8.1

